src/mobile.wiki

```diff
@@ -71,23 +71,39 @@ For all mobile platforms:
 For Android:
 
 * Install JDK 7 or newer ({{{openjdk-7-jdk}}} package in Debian/Ubuntu)
-* Install {{{qemu-kvm}}}
 * Download and untar the [[http://developer.android.com/sdk|Android SDK]]
   (the smaller version without Android Studio suffices).
 * Run {{{tools/android}}}
 * Using the Android package management interface,
   install latest versions of {{{SDK Tools}}}, {{{SDK Platform-tools}}}, and
   {{{SDK Build-tools}}}
-* Install Android API 23: {{{SDK Platform}}} and a x86-64 system image
-  (e.g., {{{Intel x86 Atom_64}}}).
+* Install Android API 23 (which corresponds to Android 6.0): {{{SDK Platform}}} .
 * From Extras, enable the {{{Android Support Repository}}} and the
   {{{Google Repository}}}.
 * For convenience, add the SDK directories {{{platform-tools}}} and
   {{{tools}}} in your {{{$PATH}}}
-* Configure an emulator: from the Android interface, go to
+
+If you will test on an Android device, you don't need to do anything else.
+
+If you want to emulate an Android device, you need to
+* install {{{qemu-kvm}}}
+* install a system image (e.g., {{{Intel x86 Atom_64}}})
+* configure an emulator: from the Android interface, go to
   {{{Tools -> Manage AVDs -> Device definition}}}
   (for example choose Nexus 6 no skin)
 
+You can install any Android API version you want, depending on the platform you
+want to target. By default, in ocsigen-start, the minimal version is Android API
+15.
+
+For iOS:
+
+* Xcode installs all dependencies you need.
+
+For Windows:
+
+* See {{{Makefile.mobile}}} in the template.
```

Comment:
Could you move the Windows instructions here? They are somewhat hidden in the Makefile :).